source/unstyled/_search_bar.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="search-bar">
+  <form role="search">
+    <input type="search" placeholder="Enter Search" />
+    <button type="submit">
+      <img src="https://raw.githubusercontent.com/thoughtbot/refills/master/source/images/search-icon-black.png" alt="Search Icon">
```

Comment:
Should this button have fallback text for the image?

If this is an un-styled component, is the image needed at all? I’d love to see a screenshot.